testing/sweeper/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=sweeper
+pkgver=19.04.2
+pkgrel=0
+arch="all"
+url="https://kde.org/applications/utilities/org.kde.sweeper"
+pkgdesc="System cleaner to help clean unwanted traces the user leaves on the system"
+license="LGPL-2.1-or-later"
+makedepends="extra-cmake-modules qt5-qtbase-dev kactivities-stats-dev kbookmarks-dev kcrash-dev kconfig-dev kconfigwidgets-dev kcoreaddons-dev kdoctools-dev ki18n-dev kio-dev ktextwidgets-dev kxmlgui-dev"
+checkdepends="xvfb-run"
+source="https://download.kde.org/stable/applications/$pkgver/src/$pkgname-$pkgver.tar.xz"
+subpackages="$pkgname-doc $pkgname-lang"
+
+build() {
+	cmake \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib
+	make
+}
+
+check() {
+	CTEST_OUTPUT_ON_FAILURE=TRUE ctest
```

Comment:
xvfb-run isn't used here (is it used implicitly during tests ?)

Reply:
Sorry my mistake, checkdepends is not needed.